src/commonTest/kotlin/fr/acinq/lightning/payment/IncomingPaymentHandlerTestsCommon.kt

```diff
@@ -302,32 +302,6 @@ class IncomingPaymentHandlerTestsCommon : LightningTestSuite() {
         assertEquals(setOf(expected), result.actions.toSet())
     }
 
-    @Test
```

Comment:
I don't think we should remove those tests, but rather setup a liquidity policy that makes us reject the corresponding pay-to-open(s)?

Reply:
Done in 8d2ff5b2. Note that the scope of those tests have changed: they now test the policy instead of the min amount (which doesn't exist anymore).